src/middlewared/middlewared/alembic/versions/22.12/2022-12-30_13-21_fix_netcli_refs.py

```diff
@@ -0,0 +1,28 @@
+"""fix netcli references
+
+Revision ID: 5cc601ce9a8e
+Revises: 136adf794fed
+Create Date: 2022-12-30 13:21:11.005256+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5cc601ce9a8e'
+down_revision = '136adf794fed'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    conn.execute(
+        'UPDATE account_bsdusers SET bsdusr_shell = "/usr/sbin/nologin" WHERE instr("bsdusr_shell", "netcli") > 1'
```

Comment:
`instr("bsdusr_shell", "netcli")` will check if `"netcli"` is in literal string `"bsdusr_shell"`which is always `0`.  You probably meant `instr(bsdusr_shell, "netcli") > 1`. Or, better, just `bsdusr_shell LIKE "%netcli%"`

Reply:
No, this works as intended

Reply:
I looked at the sqlite official documentation for `instr` and wow it's bad. I've moved to using `WHERE bsdusr_shell LIKE "%netcli%"` since it's much easier to understand.